parsely_raw_data/event.py

```diff
@@ -277,6 +298,12 @@ def to_dict(self):
             event_dict['campaign.__version__'] = self.campaign.__version__
         else:
             event_dict['campaign'] = False
+        if self.flags:
+            event_dict['flags'] = True
+            event_dict['flags.is_amp'] = self.flags.is_amp
```

Comment:
I was just trying to follow what every other sub-obj is doing here. I don't think doing `EventFlags` differently would be a good idea.